### DISCUSSION TITLE
Have Brave open new tab on homepage

### DISCUSSION THREAD

**Discussion by JackVarble** [moudey/Shell#588] [Q&A]
I tried to add an arg to brave.exe in my right-click menu, but can't figure out how to get it to always open in a new tab (not window) to the brave homepage.

`args="--new-tab 'brave://newtab'"`

Any help would be appreciated

**Comment by RubicBG** [comment]
i don't use Brave. Try
```
args="--new-tab "
// or 
args="--new-tab google.com"
```
use full path to Brave
```
cmd='C:\Program Files\BraveSoftware\Brave-Browser\Application\brave.exe'
```

**Reply by JackVarble** [reply]
Thanks for the quick response.  I do have the full path to brave in the cmd operation just didn't put it in here for security reasons.

I tried `args='--new-tab'` but it still opens the new tab in a new window not on the current window of brave.

When I do 

`args="--new-tab 'brave://newtab'"`

I get it to open a new tab in my current window, but get the error of "Can't open brave//newtab".  Is there a different syntax to tell it the ":" is actually a colon?

**Comment by RubicBG** [comment]
does with `args="--new-tab google.com"` work?
does with `args="--new-tab "` work, where space exist before quote?
i have not found such syntax in internet: `args="--new-tab 'brave://newtab'"`

**Reply by JackVarble** [reply]
Yes, the `args="--new-tab google.com"` works fine.

The `args="--new-tab "` with or without space just opens a new window.

If I do `args="--new-tab brave://newtab"` (no quotes around the address) it still just opens a new window.

The brave://newtab is the address that brave recognizes to open the new tab homepage.  I might just have to give up and have it open a new search tab or something.

**Reply by RubicBG** [reply]
try to make cmd command that works with `brave://newtab` - if you can, then NS can imitate the command, if not the is "brave problem"

**Reply by JackVarble** [reply]
It's not working, I just set it to a search engine and it's working fine.  Thanks for the tips and help.
